website/content/docs/configuration/listener/unix.mdx

```diff
@@ -0,0 +1,160 @@
+  like `"30s"` or `"1h"`.
+
+- `http_read_header_timeout` `(string: "10s")` - Specifies the amount of time
+  allowed to read request headers. This is specified using a label suffix like
+  `"30s"` or `"1h"`.
+
+- `http_read_timeout` `(string: "30s")` - Specifies the maximum duration for
+  reading the entire request, including the body. This is specified using a
+  label suffix like `"30s"` or `"1h"`.
+
+- `http_write_timeout` `string: "0")` - Specifies the maximum duration before
+  timing out writes of the response and is reset whenever a new request's header
+  is read. The default value of `"0"` means inifinity. This is specified using a
+  label suffix like `"30s"` or `"1h"`.
+
+<!-- We don't have this implemented yet
+- `max_request_size` `(int: 33554432)` – Specifies a hard maximum allowed
+  request size, in bytes. Defaults to 32 MB. Specifying a number less than or
+  equal to `0` turns off limiting altogether.
+!-->
+
+- `max_request_duration` `(string: "90s")` – Specifies the maximum
+  request duration allowed before Boundary cancels the request. This overrides
+  `default_max_request_duration` for this listener.
+
+- `tls_disable` `(string: "false")` – Specifies if TLS will be disabled. Boundary
+  assumes TLS by default, so you must explicitly disable TLS to opt-in to
+  insecure communication.
+
+- `tls_cert_file` `(string: <required-if-enabled>, reloads-on-SIGHUP)` –
+  Specifies the path to the certificate for TLS. To configure the listener to
+  use a CA certificate, concatenate the primary certificate and the CA
+  certificate together. The primary certificate should appear first in the
+  combined file. On `SIGHUP`, the path set here _at Boundary startup_ will be used
+  for reloading the certificate; modifying this value while Boundary is running
+  will have no effect for `SIGHUP`s.
+
+- `tls_key_file` `(string: <required-if-enabled>, reloads-on-SIGHUP)` –
+  Specifies the path to the private key for the certificate. If the key file
+  is encrypted, you will be prompted to enter the passphrase on server startup.
+  The passphrase must stay the same between key files when reloading your
+  configuration using `SIGHUP`. On `SIGHUP`, the path set here _at Boundary
+  startup_ will be used for reloading the certificate; modifying this value
+  while Boundary is running will have no effect for `SIGHUP`s.
+
+- `tls_min_version` `(string: "tls12")` – Specifies the minimum supported
+  version of TLS. Accepted values are "tls10", "tls11", "tls12" or "tls13".
+
+  ~> **Warning**: TLS 1.1 and lower are generally considered insecure.
+
+- `tls_cipher_suites` `(string: "")` – Specifies the list of supported
+  ciphersuites as a comma-separated-list. The list of all available ciphersuites
+  is available in the [Golang TLS documentation][golang-tls].
+
+- `tls_prefer_server_cipher_suites` `(string: "false")` – Specifies to prefer the
+  server's ciphersuite over the client ciphersuites.
```

Comment:
```suggestion
  server's ciphersuite over the client ciphersuite.
```

Reply:
The client sends a list of suites over, not just a single one.